src/components/team/Teammate.jsx

```diff
@@ -1,5 +1,34 @@
+import React from "react";
+import Image from "next/image";
+
 const Teammate = () => {
-  return <div>Teammate</div>;
+  return (
+    <div className="bg-gradient-to-r from-hlg-blue-300 to-hlg-black ...">
+      <div className="flex">
+        <div className="mr-50">
+          <Image
+            src="team/graves.png" // Path to your image inside the 'public' directory
```

Comment:
import image as an object instead of specifying height and width.

i.e. import Graves from "public/team/graves.png"
...
<Image src={Graves}